docs/ibc/overview.md

```diff
@@ -1,180 +1,150 @@
-<!--
-order: 1
--->
+<!-- order: 1 -->
 
-# Overview
+ # IBC Overview
 
-Learn what IBC is, its components and use cases. {synopsis}
+Learn what IBC is, its components, and use cases. {synopsis}
 
-## What is the Interblockchain Communication Protocol (IBC)?
+## What is the Inter-Blockchain Communication Protocol (IBC)?
 
-This document serves as a guide for developers who want to write their own Inter-blockchain
-Communication Protocol (IBC) applications for custom [use-cases](https://github.com/cosmos/ics/blob/master/ibc/4_IBC_USECASES.md).
+This document serves as a guide for developers who want to write their own IBC apps for custom use cases. <!--[use-cases](https://github.com/cosmos/ics/blob/master/ibc/4_IBC_USECASES.md)-->.
 
-Due to the modular design of the IBC protocol, IBC
-application developers do not need to concern themselves with the low-level details of clients,
-connections, and proof verification. Nevertheless a brief explanation of the lower levels of the
-stack is given so that application developers may have a high-level understanding of the IBC
-protocol. Then the document goes into detail on the abstraction layer most relevant for application
-developers (channels and ports), and describes how to define your own custom packets, and
-`IBCModule` callbacks.
+The modular design of the IBC protocol means that IBC app developers do not require in-depth knowledge of the low-level details of clients, connections, and proof verification. This brief explanation of the lower levels of the stack is provided so that app developers can gain a high-level understanding of the IBC protocol.
 
-To have your module interact over IBC you must: bind to a port(s), define your own packet data (and
-optionally acknowledgement) structs as well as how to encode/decode them, and implement the
-`IBCModule` interface. Below is a more detailed explanation of how to write an IBC application
-module correctly.
+The abstraction layer details on channels and ports are relevant for app developers. You can define your own custom packets and IBCModule callbacks.
+
+The following requirements must be met for a module to interact over IBC:
+
+- Bind to one or more ports
+
+- Define the packet data
+
+- Define optional acknowledgement structures and methods to encode and decode them
+
+- Implement the IBCModule interface
 
 ## Components Overview
 
+This section describes the IBC components.
+
 ### [Clients](https://github.com/cosmos/cosmos-sdk/tree/master/x/ibc/core/02-client)
```

Comment:
this link is not working for me... I don't see the ibc module on master (is it expected to be on master?) the target file is on https://github.com/cosmos/cosmos-sdk/tree/v0.42.3/x/ibc/core/02-client